src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
```

Comment:
What this `100` mean? To improve readabillity please extract this value to constant variable on class level and name it properly to describe what the value is. Keep in mind constant naming convention
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names